.github/workflows/ci_test.yml

```diff
@@ -1,5 +1,7 @@
+---
 name: GitHub CI
 
+# yamllint disable-line rule:truthy
```

Comment:
Let's disable truthy in the `.yamllint` file - I would prefer not having to add yamllint comments in our yaml files.